packages/typescript-configs/core/base.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "compileOnSave": false,
```

Comment:
What is this config value? I can't find it on https://www.typescriptlang.org/tsconfig